ansible/roles/etcd/tasks/etcd2-configure.yml

```diff
@@ -11,10 +11,3 @@
 - name: CoreOS | Write etcd config file
   template: src=etcd.conf.j2 dest=/etc/etcd/etcd.conf
   notify: restart etcd2
```

Comment:
Based on this [1], restart etcd2 should not work as etcd2_started is not defined any more in this task block. From ``ansible/roles/etcd/handlers/main.yml +11``:

```
- name: restart etcd2
  service: name=etcd2 state=restarted
  when: etcd2_started.changed == false
```

[1] http://docs.ansible.com/ansible/playbooks_intro.html#handlers-running-operations-on-change